infra/modules/azure_service_bus_alerts/variables.tf

```diff
@@ -0,0 +1,108 @@
+variable "environment" {
+  type = object({
+    prefix          = string
+    env_short       = string
+    location        = string
+    domain          = optional(string)
+    app_name        = string
+    instance_number = string
+  })
+
+  description = "Values which are used to generate resource names and location short names. They are all mandatory except for domain, which should not be used only in the case of a resource used by multiple domains."
+}
+
+variable "alerts_on_active_messages" {
+  type = object({
+    description     = string
+    entity_names    = list(string)
+    threshold       = optional(number, 10)
+    severity        = optional(string, "Warning")
+    auto_mitigate   = optional(bool, true)
+    check_every     = optional(string, "PT15M")
+    lookback_period = optional(string, "PT30M")
+    enable          = optional(bool, true)
+  })
+
+  default     = null
+  description = <<EOF
+"Configure alert over messages moved in dead-letter queue. If set to null, no alert will be created
```

Comment:
question: what's the diffence vs `alerts_on_dlq_messages` ?

Reply:
It's not clear from the description (which is the same) when to use one vs the other.

Reply:
Updated